src/TranslationFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend;
+
+use Symfony\Component\Translation\Loader\JsonFileLoader;
+use Symfony\Component\Translation\Translator;
+use Symfony\Component\Translation\MessageSelector;
+
+/**
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class TranslationFactory {
```

Comment:
I wanted the factories to be testable in cases where they do more than just create a new instance. 
Also, I did not want to make `FunFunFactory` bigger.